packages/pagination/src/PaginationControls.js

```diff
@@ -161,5 +174,7 @@ PaginationControls.defaultProps = {
   pageRange: 5,
   marginPages: 2,
   breakLabel: true,
+  ariaLabel: 'pagination',
```

Comment:
We should remove all remaining references to ariaLabel since it's no longer used (and aria-label is covered by the {...rest} object). This would include removing it from the docs and storybook.

Reply:
@LauRoxx done